code/backend/batch/utilities/helpers/config/config_helper.py

```diff
@@ -184,6 +191,12 @@ def get_active_config_or_default():
 
         return Config(config)
 
+    @staticmethod
+    @functools.cache
+    def get_default_assistant_prompt():
+        config = ConfigHelper.get_default_config()
+        return  config["prompts"]["answering_user_prompt"]
```

Comment:
Should this be: return config.get("answering_user_prompt")?

Reply:
I think that can be both, but it should be `config["prompts"].get("ai_assistant_type")`. 
let me know if that is a good answer for you or I change the line =)